index.xml

```diff
@@ -9809,9 +9755,8 @@ counter.value;                                           <span class='comment'>/
               <a class='dfnref' href='#dfn-extended-attribute'>extended attribute</a>
               appears on an <a class='dfnref' href='#dfn-interface'>interface</a>,
               <a class='dfnref' href='#dfn-partial-interface'>partial interface</a>,
-              an individual <a class='dfnref' href='#dfn-interface-member'>interface member</a>, or
-              <a class='dfnref' href='#dfn-dictionary'>dictionary</a> with a <a href='#es-dictionary-constructors'>constructor</a>,
-              it indicates that the interface, interface member or <a href='#es-dictionary-constructors'>dictionary constructor</a> is exposed
+              an individual <a class='dfnref' href='#dfn-interface-member'>interface member</a>,
+              it indicates that the interface or interface member is exposed
               only within a
               <a href='https://w3c.github.io/webappsec-secure-contexts/#secure-context'>secure context</a>
               (<a href='#ref-SECURE-CONTEXTS'>[SECURE-CONTEXTS]</a>, section 2).
```

Comment:
Two paragraphs below here there is a reference to dictionaries that should be removed.